### PR TITLE
FIX: Update fury to 0.6.*

### PR DESCRIPTION
**Pull request by AntoineTheb**
Self explanatory.

You can test that fury still works and displays streamlines by running `scil_visualize_bundle_mosaic.py` with the data here:
https://drive.google.com/drive/folders/1qgnXRayGJGPDOwuzqeC6clZWbOgc2b8x?usp=sharing

### CODE REVIEW

File: scripts/scil_visualize_seeds.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def main():
         raise ValueError("Invalid input streamline file format " +
                          "(must be trk): {0}".format(args.tractogram_filename))
 
-    # Load files and data. TRKs can have 'same' as reference
+    # Load files and data. TRKs can have 'same' as refescenece
```
Comment (CHrlS98) [inline scripts/scil_visualize_seeds.py:49]: Typo, right?

**Review by CHrlS98 — COMMENTED** [review]
Except for the typo, it all looks good to me!

File: scripts/scil_visualize_seeds.py (hunk 1 above)
Comment (frheault) [reply scripts/scil_visualize_seeds.py:49]: If it is about the Mediterranean recluse spider, it is rufescens, not refescenece ;)

Comment (AntoineTheb) [reply scripts/scil_visualize_seeds.py:49]: Ha ! My regex might have been too permissive. I'll fix that asap.

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**